__tests__/app.test.js

```diff
@@ -379,3 +378,31 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+    test("200 - responds with an array of user objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          const { users } = body;
+          expect(users).toHaveLength(4);
+          users.forEach((user) => {
+            expect(user).toMatchObject({
+              username: expect.any(String),
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+            });
+          });
+        });
+    });
+  
+    test('404 - responds with "Resource Not Found" when requesting an invalid endpoint resource', () => {
```

Comment:
This test is fine but isn't specific to api/users and relates to the whole api. It should therefore be in it's own describe block for generic 404 errors 👍